### PR TITLE
react-stopwatch-component | Lucca Rizzi

### PR DESCRIPTION
**Pull request by luccarizzi**
new1

### CODE REVIEW

File: react-stopwatch-component/src/stopwatch.jsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      control: false,
+      count: 0,
+      watch: ''
+    };
+    this.pausePlay = this.pausePlay.bind(this);
+    this.tick = this.tick.bind(this);
+    this.reset = this.reset.bind(this);
+  }
+
+  render() {
```
Comment (Zhiwutian) [inline react-stopwatch-component/src/stopwatch.jsx:16]: it is generally considered a good idea to keep the render method as the last method in the class.

File: react-stopwatch-component/src/stopwatch.jsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+class Stopwatch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      control: false,
```
Comment (Zhiwutian) [inline react-stopwatch-component/src/stopwatch.jsx:7]: I would remove this from state, and use the `watch` property to tell if there is currently an interval going. if one is running you can reset the value to an empty string or other falsy value. A trim state is a happy state.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
a couple small updates!

File: react-stopwatch-component/src/stopwatch.jsx (hunk 1 above)
Comment (luccarizzi) [reply react-stopwatch-component/src/stopwatch.jsx:16]: Done.

File: react-stopwatch-component/src/stopwatch.jsx (hunk 2 above)
Comment (luccarizzi) [reply react-stopwatch-component/src/stopwatch.jsx]: Done.

**Review by Zhiwutian — APPROVED** [review]
👍

**Pull request merged**